core/modules/wmgr/python/dbMgr.py

```diff
@@ -916,7 +902,7 @@ def getIndex(dbName, tblName, chunkId=None):
         allRows = dbConn.execute(query)
 
         if allRows.keys():
-            descr = [dict(name=d[0], type=_typeCode2Name(d[1])) for d in allRows.keys()]
+            descr = [dict(name=d[0], type=utils.typeCode2Name(dbConn, d[1])) for d in allRows.keys()]
```

Comment:
This is totally broken, it was actully broken by previous commit which I failed to notice at that time. Old pre-sqlalchemy code used `cursor.description` to get type codes here, `keys()` only return the list of column names. `d[1]` above returns second character of the column name which is not a type code. We need to fix this, and this should be related to what Brian was doing with dax_dbserv when it needs to return column types. Maybe we should factor out Brian's code and move it to db.utils, for now I think it should be fie to replace `allRows.keys()` with `allRows.cursor.description`.